doc/user/content/sql/create-sink.md

```diff
@@ -163,6 +165,22 @@ You can find the topic name for each Kafka sink by querying `mz_kafka_sinks`.
 
 {{% kafka-sink-drop  %}}
 
+#### Enabling topic reuse after restart
+
+{{< version-added v0.8.2 />}}
+
+This feature is only available in experimental mode. By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, when you create a sink, you must:
+
+* Enable the `exactly_once` switch.
+* Specify a consistency topic to store the information Materialize will use to identify where synching left off. For more information, see [Consistency metadata](#consistency-metadata).
+
+To synchronize this history, Materialize must be able to reconstruct the history of the sinked object and all objects on which it is dependent. For Kafka, this means that the sinked topic should be set to infinite retention. Additionally, the name of the consistency_topic must be unique across all your sinks (not only the sink for which it tracks consistency data).
```

Comment:
I would leave out "To synchronize this history, ".

I think we are conflating three distinct things here.

1. All sources that the object being sinked transitively depends on need to have replayable timestamps -- meaning that currently it only works for Kafka and file sources and views based on them.

2. The sink topic name and the sink consistency topic name needs to be unique across all sinks in the system.

3. The sink topic and the sink consistency topic cannot be written to by any other process, and both need to be set to infinite data retention

Reply:
@aljoscha does the above sound accurate? also point two strikes me as potentially overly restrictive in a world where there are many users using the same mz instance

Reply:
That sounds correct, yes! Though aren't 2. and 3. the same? Also, I think that's just what you get when multiple users are writing to the same Kafka cluster. Also, I think it's fine if another process writes to the output topic, only the consistency topic shouldn't be messed with.

Reply:
not exactly -- 2 is trying to say "no other mz sink can write to data or consistency topics" and 3 is trying to say "also no other process in your system can write to those topics, nor can kafka reclaim them"

Reply:
Per @aljoscha, I modified this to say that only the consistency topic can't be written to.